src/components/Calendar/Section.js

```diff
@@ -36,28 +38,69 @@ export const styles = {
     justifyContent: 'space-between',
     flexWrap: 'wrap',
   },
+  modal: {
+    position: 'absolute',
+    width: '80%',
+    outline: 'none',
+    top: '12.5%',
+    left: '12.5%',
+    padding: '10px',
+  },
 };
 
-function Section({ classes, section }) {
-  return (
-    <Paper className={classes.paper}>
-      <div className={classes.container}>
-        <div className={classes.header}>
-          <Typography variant="caption" className={classes.text}>
-            {getFormattedClassEvent(section.event)}
-          </Typography>
+class Section extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+    };
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  toggleModal() {
+    const { showModal } = this.state;
+    this.setState({
+      showModal: !showModal,
+    });
+  }
+
+  render() {
+    const { classes, section } = this.props;
+    const { showModal } = this.state;
+    return (
+      <div>
+        <Paper className={classes.paper} onClick={this.toggleModal}>
+          <div className={classes.container}>
+            <div className={classes.header}>
+              <Typography variant="caption" className={classes.text}>
+                {getFormattedClassEvent(section.event)}
+              </Typography>
 
-          <Typography variant="caption" className={classes.text}>
-            {`${section.subjectId} ${section.courseId}`}
-          </Typography>
-        </div>
+              <Typography variant="caption" className={classes.text}>
+                {`${section.subjectId} ${section.courseId}`}
+              </Typography>
+            </div>
 
-        <Typography variant="subtitle2" className={`${classes.text} ${classes.name}`} noWrap>
-          {section.name}
-        </Typography>
+            <Typography variant="subtitle2" className={`${classes.text} ${classes.name}`} noWrap>
+              {section.name}
+            </Typography>
+          </div>
+        </Paper>
+        <Modal
+          aria-labelledby="simple-modal-title"
+          aria-describedby="simple-modal-description"
+          open={showModal}
+          onClose={this.toggleModal}
+        >
+          <Paper className={classes.modal}>
+            <Typography variant="h5">
+              {`${section.subjectId} ${section.courseId} -- Section ${section.sectionNumber}`}
```

Comment:
I know this is probably directly from 2.0 but let's change this to "EECS 101-0 Section 21" instead (so no more `--`.